libsrc/_DEVELOPMENT/EXAMPLES/benchmarks/fasta/z88dk-new/readme.txt

```diff
@@ -9,11 +9,14 @@ VERIFY CORRECT RESULT
 To verify the correct result compile for the zx spectrum target
```

Comment:
CP/M target

Reply:
I used CP/M to test for correctness on real target, and this slipped through. Will fix, along with another typo I noticed elsewhere.